wear/build.gradle

```diff
@@ -0,0 +1,68 @@
+plugins {
+    id 'com.android.application'
+    id 'org.jetbrains.kotlin.android'
+}
+
+android {
+    namespace 'com.android.wear'
+    compileSdk 33
+
+    defaultConfig {
+        applicationId "com.android.wear"
```

Comment:
`com.android.gpstest.wear`

Reply:
> `com.android.gpstest.wear`

Ok, I will also set the namespace to the same value to coordinate.